lib/getUniqueId.js

```diff
@@ -0,0 +1,63 @@
+'use strict';
+var Cesium = require('cesium');
+var defined = Cesium.defined;
+
+module.exports = getUniqueId;
+
+// Given a prefix for a new ID, checks the json for matching prefixes throughout and generates a unique ID.
+function getUniqueId(json, prefix) {
```

Comment:
`json` should be `gltf` or whatever we call it throughout this project.  `json` is too generic.